plugins/modules/dcnm_policy.py

```diff
@@ -44,6 +44,13 @@
       - query
     default: merged
 
+  use_desc_as_key:
+    description:
+    - A flag specifying whether using the description as unique key for policies.
```

Comment:
Suggest rewording this a bit

`Flag to enforce using the description parameter as the unique key for policy management"

Reply:
Done